scripts/test

```diff
@@ -1,41 +1,37 @@
 #!/bin/bash
 set -e
 
+CWD=$(pwd -P)
 cleanup()
 {
     EXIT=$?
-    kill $PID
+    set +e
+    echo Stopping rancher server
+    kill %%
+    wait
```

Comment:
Use the PID that's going to feed `kill` here

Reply:
Updated